core/domain/acl_decorators.py

```diff
@@ -25,6 +25,23 @@
 current_user_services = models.Registry.import_current_user_services()
 
 
+def _is_activity_public(activity_rights):
+    """Checks whether activity is in public domain.
```

Comment:
"Public" !== "public domain". Change to:

Checks whether an activity has been published.

Better still: implement this as a method on the rights object. Maybe has_been_published().

(Note that we are planning to deprecate the "publicized" functionality soon btw.)